specification/cognitiveservices/data-plane/ContentSafety/preview/2023-10-15-preview/contentsafety.json

```diff
@@ -703,33 +703,29 @@
           }
         },
         "outputType": {
-          "$ref": "#/definitions/AnalyzeImageOutputType",
+          "type": "string",
           "description": "This refers to the type of image analysis output. If no value is assigned, the default value will be \"FourSeverityLevels\".",
-          "default": "FourSeverityLevels"
+          "default": "FourSeverityLevels",
+          "enum": [
+            "FourSeverityLevels"
+          ],
+          "x-ms-enum": {
+            "name": "AnalyzeImageOutputType",
+            "modelAsString": true,
+            "values": [
+              {
+                "name": "FourSeverityLevels",
+                "value": "FourSeverityLevels",
+                "description": "Output severities in four levels, the value could be 0,2,4,6."
+              }
+            ]
+          }
         }
       },
       "required": [
         "image"
       ]
     },
-    "AnalyzeImageOutputType": {
```

Comment:
@timotheeguerin, @markcowl: I think this is technically a breaking change, triggered by specs with `omit-unreachable-types: true`.  Is the breaking change acceptable?

Reply:
It looks like the new TypeSpec will "inline" definitions in some cases where previously they were in "definitions". This is definitely not breaking at the REST API level, and the SDK generators _should_ be able to handle this. I'll add the BreakingChanges-Approved label.

Reply:
Yeah the enum is still there just inline

Reply:
Yeah, autorest will emit a single enum for each of these inline enums, using the x-ms-enum name specified